### ISSUE TITLE
Special announce interrupt not working?

### ISSUE THREAD

**Issue by moni-git-1** [DeadlyBossMods/DeadlyBossMods#1291]
**Describe the bug**
Not getting special announce for interrupting Enfeebling Spittle in Zekvir's Lair.

**Do you have an error log of what happened?**
No errors, all other special announces seem to work well for this encounter and others.

**To Reproduce**
Enter Zekvir's Lair in ? difficulty with configuration for Enfeebling Spittle "Show special announce (with count) to interrupt" checked. I'm a devastation evoker if that helps.

**Screenshots**

![image](https://github.com/user-attachments/assets/1a914750-46fc-4d63-bdfa-bd96281aa243)

**Did you try having DeadlyBossMods as the only enabled addon and everything else (especially something like ElvUI) disabled?**
Yes

**Which version of DeadlyBossMods are you using?**
11.0.19

**Was it working in a previous version? If yes, which was the last good one?**
Don't know, first time trying

**Additional context**
N/A

**Comment by MysticalOS** [comment]
do you have the dungeon update?

**Comment by moni-git-1** [comment]
Is this right?

![image](https://github.com/user-attachments/assets/b8164cec-5f58-4c05-a16d-1c78e7ff37a8)

**Comment by MysticalOS** [comment]
yes, that should work, unless there are any other new spellids i missed they hotfixed in. that added the new Id I saw 2 days ago

**Comment by moni-git-1** [comment]
Just tested again and still not working, unfortunately. Is there something I might be missing in configuration?

**Comment by MysticalOS** [comment]
Is your interrupt on cooldown? it won't tell you to interrupt it if it is.

**Comment by moni-git-1** [comment]
Nope, this happens at the first cast even before I've used my interrupt.

**Comment by moni-git-1** [comment]
Is there anything else I can try here? I'd really like to beat Zekvir, but I'm just learning how to interrupt and struggling without a reminder on what feels like a really short cast. It's weird because if I check "dispel" special announce, I do hear the sound, but it's just for "interrupt" that I don't, even if my interrupt ability has not been used. :(

**Comment by MysticalOS** [comment]
no idea. if you can provide a log of it i can see what spellid is used for your version of fight. blizzard made a mess of things because they made every class now face a diff version of fight. you might not get same spell ids I do

**Comment by moni-git-1** [comment]
Sure, I can do that! How do I get a log?

**Comment by MysticalOS** [comment]
/combatlog

then after at least one pull, log out. go into your wow folder/logs and look for wowcombat or something of that nature. then drop that file here.

**Comment by moni-git-1** [comment]
[WoWCombatLog-093024_211740.txt](https://github.com/user-attachments/files/17199346/WoWCombatLog-093024_211740.txt)

**Comment by moni-git-1** [comment]
Is this what you're looking for?

SPELL_CAST_START,Creature-0-3885-2682-27400-225204-00007B7754,"Zekvir",0x10a48,0x0,0000000000000000,nil,0x80000000,0x80000000,450505,"Enfeebling Spittle",0x8

**Comment by MysticalOS** [comment]
that's helpful. I have correct Id in place but I may have spotted one reason the interrupt warning didn't work. the backup non interrupt warning would have still been functional though, but i saw in your screen shot you turned backup alert off

new alpha going out soon should fix main interrupt warning, but backup alert also ideally should stay enabled cause that fires when interrupt is on cooldown.

**Comment by moni-git-1** [comment]
Ohhh "Announce when Enfeebling Spittle has been cast" is what you're talking about, right? The default sound is subtle but now I hear it. And it's happening at the right time for me to try to interrupt. Playing around with it now to see if I can change the sound to something a bit more AAH. Thank you so much!

**Comment by MysticalOS** [comment]
well the emphasized warning is fixed too but you gotta use dungeon alpha package for now.

**Comment by moni-git-1** [comment]
No need! <3

![image](https://github.com/user-attachments/assets/6b0afd5b-266b-44bd-b881-1617b1e65462)

I changed this, and I was finally able to do it first try!
![image](https://github.com/user-attachments/assets/f8e3844d-53d7-4860-b3d2-6019d5d4fc79)

**Comment by MysticalOS** [comment]
oof, don't leave that setting. it'll be very bad in most locations :D
